CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Changed
+
+### Fixed
+
+[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.6.0-rc.0...main
+
+## [v2.6.0-rc.0][v2.6.0-rc.0]
```

Comment:
@sumo-drosiek I think we established that we don't include pre releases in changelog.

Reply:
commit removed